js/src/personal/bryan/About Me/AboutMe.page.tsx

```diff
@@ -0,0 +1,25 @@
+import { Title, Group } from '@mantine/core'
+import classes from '../Bryan.module.css'
```

Comment:
nit: try to use the absolute paths like '@/.../personal/bryan/...'

Absolute paths are easier to refactor and search for